syntax = "proto3";

package wxz_rpc;

message header{
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 args_size = 3;
}